Neos.Media.Browser/Configuration/Policy.yaml

```diff
@@ -29,6 +29,9 @@ privilegeTargets:
       matcher: 'method(Neos\Media\Browser\Controller\(Asset|Image)Controller->(replaceAssetResource|updateAssetResource)Action())'
 
 roles:
+  'Neos.Media.Browser:WorkspaceName':
```

Comment:
If we just assign the privilege to the admin, we don't need this role IMO. If someone needs it, they can create their own.